### PR TITLE
add test to inspect

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: tests/inspect.bats [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 load helpers
 
+@test "inspect" {
+	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+	run buildah commit --signature-policy ${TESTSDIR}/policy.json "$cid" alpine-image
+	[ "$status" -eq "0" ]
+	run buildah inspect alpine
```
Comment (rhatdan) [inline tests/inspect.bats]: Would it be possible to check some fields in these images to make sure they are the same.
Would probably make for  a better tests.

Check Cmd, Entrypoint, Env, Labels ...

Comment (q384566678) [reply tests/inspect.bats]: updated, WDYT?

**Comment by q384566678** [comment]
updated.

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green tests buttons.
If at some point you could add a 'buildah config' command to this or a new test to fill in some fields like author, entrypoint, cmd, etc and then test that they are filled with the values you expect, that's would be great.

**Comment by rhatdan** [comment]
Awesome @q384566678 
Adding tests is something Engineers usually are reticent to do, but really help out in the long run.  I am as guilty as anyone of this.
@rh-atomic-bot r+

**Comment by q384566678** [comment]
>If at some point you could add a 'buildah config' command to this or a new test to fill in some fields like author, entrypoint, cmd, etc and then test that they are filled with the values you expect, that's would be great.

I thought about it at first, but I think this kind of test has been implemented in config.bats. It feels repetitive when you reuse it. I don't know if my understanding is correct.

**Pull request closed without merging**